src/main/java/seedu/address/model/person/Faculty.java

```diff
@@ -83,4 +85,13 @@ public boolean equals(Object other) {
     public int hashCode() {
         return studentFaculty.hashCode();
     }
+
+    /**
+     * Returns a list of strings with enum values from the Nus enum class.
+     *
+     * @return List of Nus enum values.
+     */
+    public static List<String> getFacultyEnumAsList() {
+        return Stream.of(Nus.values()).map(Enum::name).collect(Collectors.toList());
```

Comment:
great method! my only suggestion is to maybe split up the method to avoid having complicated expressions for readability 👍

Reply:
As this step is to convert a enum class to a list of strings, I believe that doing it in one line makes it clearer instead to separating it into different lines of stream type and may make it difficult to understand too.

Reply:
sure thing good job jia le 👍